tensorflow_probability/python/bijectors/gev_cdf.py

```diff
@@ -0,0 +1,218 @@
+
+from tensorflow_probability.python.bijectors import bijector
+from tensorflow_probability.python.internal import assert_util
+from tensorflow_probability.python.internal import dtype_util
+from tensorflow_probability.python.internal import tensor_util
+
+
+__all__ = [
+    'GeneralizedExtremeValueCDF',
+]
+
+
+class GeneralizedExtremeValueCDF(bijector.Bijector):
+  """Compute `Y = g(X) = exp(-t(X))`, the GeneralizedExtremeValue CDF,
+  where `t(x)` is defined to be:
+    *`(1 + conc * (x - loc) / scale) ) ** (-1 / conc)` when `conc != 0`;
+    *`exp(-(x - loc) / scale)` when `conc = 0`.
+
+  This bijector maps inputs from the domain to `[0, 1]`, where the domain is
+    * [loc - scale/conc, inf) when conc > 0;
+    * (-inf, loc - scale/conc] when conc < 0;
+    * (-inf, inf) when conc = 0;
+
+  The inverse of the bijector applied to a uniform random variable
+  `X ~ U(0, 1)` gives back a random variable with the
+  [Generalized extreme value distribution](
+  https://https://en.wikipedia.org/wiki/Generalized_extreme_value_distribution):
+
+  ```none
+  Y ~ GeneralizedExtremeValueCDF(loc, scale, conc)
+  pdf(y; loc, scale, conc) = t(y; loc, scale, conc) ** (1 + conc) * exp(
+  - t(y; loc, scale, conc) ) / scale
+  where t(x) =
+    * (1 + conc * (x - loc) / scale) ) ** (-1 / conc) when conc != 0;
+    * exp(-(x - loc) / scale) when conc = 0.
+  ```
+  """
+
+  def __init__(self,
+               loc=0.,
+               scale=1.,
+               concentration=0,
+               validate_args=False,
+               name='generalizedextremevalue_cdf'):
+    """Instantiates the `GeneralizedExtremeValueCDF` bijector.
+
+    Args:
+      loc: Float-like `Tensor` that is the same dtype and is
+        broadcastable with `scale` and `concentration`.
+      scale: Positive Float-like `Tensor` that is the same dtype and is
+        broadcastable with `loc` and `concentration`.
+      concentration: Nonzero float-like `Tensor` that is the same dtype and is
+        broadcastable with `loc` and `scale`.
+      validate_args: Python `bool` indicating whether arguments should be
+        checked for correctness.
+      name: Python `str` name given to ops managed by this object.
+    """
+    parameters = dict(locals())
+    with tf.name_scope(name) as name:
+      dtype = dtype_util.common_dtype(
+          [loc, scale, concentration], dtype_hint=tf.float32)
+      self._loc = tensor_util.convert_nonref_to_tensor(
+          loc, dtype=dtype, name='loc')
+      self._scale = tensor_util.convert_nonref_to_tensor(
+          scale, dtype=dtype, name='scale')
+      self._concentration = tensor_util.convert_nonref_to_tensor(
+          concentration, dtype=dtype, name='concentration')
+      super(GeneralizedExtremeValueCDF, self).__init__(
+          validate_args=validate_args,
+          forward_min_event_ndims=0,
+          parameters=parameters,
+          name=name)
+
+  @property
+  def loc(self):
+    """
+    The `loc` in `Y = exp(-t(X))`
```

Comment:
python docs have a 1 line doc for the first line. Can you try to condense the description to a one line doc (or a one line doc, followed by more explanation)? Here and below